src/js/player.js

```diff
@@ -782,6 +783,38 @@ class Player extends Component {
     return el;
   }
 
+  /**
+   * Get or set the `Player`'s crossOrigin option. For the HTML5 player, this
+   * sets the `crossorigin` attribute on the `<video>` tag to control the CORS
+   * behavior.
+   *
+   * @see [Video Element Attributes]{@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-crossorigin}
+   *
+   * @param {string} [value]
+   *        The value to set the `Player`'s crossorigin to. If an argument is
+   *        given, must be one of `anonymous` or `use-credentials`.
+   *
+   *
+   * @return {string}
+   *         The current crossorigin value of the `Player`.
+   */
+  crossOrigin(value) {
+    if (!value) {
+      return this.techGet_('crossorigin');
+    }
+
+    const validCorsValues = ['anonymous', 'use-credentials'];
+
+    if (validCorsValues.indexOf(value) < 0) {
+      log.error(`Improper value "${value}" supplied for crossorigin`);
+      return;
+    }
+
+    this.techCall_('setCrossorigin', value);
+
+    return this.techGet_('crossorigin');
```

Comment:
our other setters don't return a value when set, only when get.
```suggestion
    return;
```